Justfile

```diff
@@ -133,15 +193,7 @@ alias pc := pre-commit-run
     rm -rf *.egg-info
     rm -rf .venv
     rm -rf {{py_package_name}}/__pycache__/
-# Install Sphinx and any necessary extensions
```

Comment:
@Tatianah3 this shouldn't be deleted, it's both used and unrelated to the issue.

Reply:
@Tatianah3  you didn't reply to the comment